apps/src/weblab2/CDOIDE/utils/prettify.tsx

```diff
@@ -0,0 +1,35 @@
+// This whole thing is disabled - we need prettier ~3.2.5.
+
+/* tslint:disable-next-line */
+export const prettify = async (val: string, language: string) => {
```

Comment:
Let's remove, stash the code somewhere if needed (ticket? another branch?), and file a follow-up ticket.